demo/src/source-info.js

```diff
@@ -24,7 +27,22 @@ const zarrInfo = {
   url: rootZarrUrl
 };
 
+const rootStaticZarrUrl =
+  'https://vitessce-data.storage.googleapis.com/0.0.24/master_release/spraggins/spraggins.ims.zarr';
+
+const staticInfo = {
+  isPublic: false,
```

Comment:
Nils said not to make this demo public since it will just go into vitessce anyway.